mibs/lenovoxcc/lnvgy_fw_xcc_tei3c0n-4.83_lenovoproducts-anyos_noarch.mib

```diff
@@ -0,0 +1,99 @@
+LENOVO-PRODUCTS-MIB DEFINITIONS ::= BEGIN

```

Comment:
Please rename this file as 
`LENOVO-PRODUCTS-MIB`
(without any extension). General naming rule is to name the file exactly as the MIB is defined inside the file
`LENOVO-PRODUCTS-MIB DEFINITIONS ::= BEGIN`